polaris-migrator/src/utilities/sass.ts

```diff
@@ -0,0 +1,31 @@
+import type {Node, FunctionNode} from 'postcss-value-parser';
+
+function getNamespace(options?: NamespaceOptions) {
+  return options?.namespace || '';
+}
+
+export interface NamespaceOptions {
+  namespace?: string;
+}
+
+export function namespace(name: string, options?: NamespaceOptions) {
+  const namespace = getNamespace(options);
+  return namespace ? `${namespace}.${name}` : name;
+}
+
+/**
+ * @example
+ * const spacingFunction = namespace('spacing', options);
+ * const remFunction = namespace('rem', options);
+ *
+ * const isSpacingFunction = createIsSassFunction(spacingFunction);
+ * const isRemFunction = createIsSassFunction(remFunction);
+ * const isCalcFunction = createIsSassFunction('calc');
+ *
+ * if (isSpacingFunction(node)) node // FunctionNode
+ */
+export function createIsSassFunction(name: string) {
```

Comment:
NICE! Thank you for abstracting this! I knew we'd want to make this option available for all Sass migrations.

Might be good to add this and the `decl[processed]` to the Sass template

Reply:
This is a great idea and will address in a follow up PR! 🎉